app/js/components/manageApps/ManageApps.jsx

```diff
@@ -682,7 +682,8 @@ export default class ManageApps extends React.Component {
       });
     }).catch(
       (error) => {
-        toastr.error(error);
+        toastr.error("There is no internet connection");
```

Comment:
Does it mean the `get` request can only 'fail' (hence the catch block getting executed) when there's no internet connection? An error response, say 500 from the server(meaning there was internet connection to the server) could still have the error handling invoked. Consider displaying the exact response from the server to avoid confusion from users.

Reply:
Thanks @kodero 
I'm looking into this